StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButton.swift

```diff
@@ -367,13 +353,28 @@ class RowButton: UIView {
             setContentViewAlpha(1.0)
         }
     }
+
+    func makeSameHeightAsOtherRowButtonsIfNecessary() {
+        // To make all RowButtons the same height, set our height to the tallest variant (a RowButton w/ text and subtext)
+        // Don't do this if we are flat_with_checkmark style and have an accessory view - this row button is allowed to be taller than the rest
+        let displayingRightAccessoryView = !(rightAccessoryView?.isHidden ?? true)
+        guard !(isFlatWithCheckmarkStyle && displayingRightAccessoryView) else {
+            return
+        }
```

Comment:
I find this code hard to reason about what it's doing, maybe we can do:
```swift
let displayingRightAccessoryView = rightAccessoryView?.isHidden == false 

if isFlatWithCheckmarkStyle && displayingRightAccessoryView {
    return
}
```

It's nice the guard enforces that we return, but it makes it a bit tough to grok imo.